src/main/java/org/kaazing/netx/ws/internal/WebSocketTransition.java

```diff
@@ -17,6 +17,8 @@
 
 public enum WebSocketTransition {
     RECEIVED_UPGRADE_RESPONSE,
+    RECEIVED_CONTINUATION_FRAME,
+    SEND_CONTINUATION_FRAME,
     RECEIVED_CLOSE_FRAME,
     SEND_CLOSE_FRAME,
     RECEIVED_PING_FRAME,
```

Comment:
Agree.

RECEIVE/SEND for transitions and RECEIVED/SENT for states seems reasonable.

Note: RECEIVING/SENDING sounds a bit more like a state instead of an action
that triggers a transition.

Just my 2c. 😄


-- 

*John Fallows*
CTO*  |  *Phone: +1.650.960.8148  |  Mobile: +1.415.215.6597
*----------------------------------------------------------------------*
KAAZING >|<
Event-Driven Communication. Fast. Secure. Mobile. Guaranteed.
Kaazing.com <http://www.kaazing.com/>  |  Download
<http://kaazing.com/download.html>  |  Blog <http://blog.kaazing.com/>  |
Twitter <https://twitter.com/kaazing>